### ISSUE TITLE
Override naive detection of localisedstring in terms

### ISSUE THREAD

**Issue by opoudjis** [riboseinc/asciidoctor-gb#9]
`=== CVE漏洞 CVE vulnerability` will not be parsed correctly with  CVE漏洞 as Chinese; need override.

**Comment by opoudjis** [comment]
[zh]#...# can't embed within [alt]#...# after all; nor can any other formatting. Should make [alt]#...# be macros instead.

**Comment by opoudjis** [comment]
The alt#...# is now globally alt:[...]. Unfortunately Asciidoc does not parse embedded macros correctly; I wanted to have alt:[en:[stem:[t_90] vulnerability] zh:[stem:[t_90]漏洞]], for example, but Asciidoc mangles the nesting of the macros—it does not detect the end of the nested macro correctly. 

That means we still need to use [zh]#...# and [en]#....#  optionally if we want to have script-mixed terms, but when we do, those terms cannot have any further formatting embedded. (Including stem terms.) This is not great, but given that it is an edge case and I've already spent a massive amount of time on it, I've just documented it in the Readme as a known issue.

I have updated the private GB standard to deal with the script mixing of CVE.
